packages/expo-av/ios/EXAV/EXAV.m

```diff
@@ -269,7 +269,7 @@ - (NSError *)promoteAudioSessionIfNecessary
   if (!_audioIsEnabled) {
     return UMErrorWithMessage(@"Expo Audio is disabled, so the audio session could not be activated.");
   }
-  if (_isBackgrounded && ![_kernelAudioSessionManagerDelegate isActiveForModule:self]) {
+  if (_isBackgrounded && !_staysActiveInBackground && ![_kernelAudioSessionManagerDelegate isActiveForModule:self]) {
```

Comment:
I think `[_kernelAudioSessionManagerDelegate isActiveForModule:self]`  would return `true` if `_staysActiveInBackground` would be set to `true`, because we wouldn't ever inform the delegate that we are backgrounding. See

https://github.com/expo/expo/blob/87558b0d6314311a34c9a3a55b68b9a6ac831d7f/packages/expo-av/ios/EXAV/EXAV.m#L129-L140

Are you sure this change is needed for background playback to work?